scipy/optimize/_linprog.py

```diff
@@ -789,9 +789,21 @@ def _linprog_simplex(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     else:
         # Failure to find a feasible starting point
         status = 2
+        message = (
```

Comment:
This message is much better than what I wrote!
Only suggestion - and this is nitpicky and subjective, so you decide for yourself - is to set `messages[status]` to this string rather than setting `message` directly. Then you can get rid of the line you had to add `if not message` and that extra level of indentation that goes with it. Finally, since it would be a single line change, it's easier to see that it won't break anything else.